draft-ietf-aipref-vocab.md

```diff
@@ -216,6 +220,7 @@ where other priorities could override specific preferences:
   in order to develop or use tools that detect harmful content
   according to established terms of use.
 
+Because enforcement is not provided by this specification, the consequences of ignoring preferences may vary depending upon how a given legal jurisdiction recognizes preferences, including how it maps categories of use to specific applications.
```

Comment:
```suggestion
Because enforcement is not provided by this specification,
the consequences of ignoring preferences may vary
depending upon how a given legal jurisdiction recognizes preferences;
see {{applicability}}.
```